src/configs/base_keyspecs.c4m

```diff
@@ -5662,6 +5662,20 @@ AWS where imdsv2 is available.
 """
 }
 
+keyspec _NETWORK_PARTIAL_TRACEROUTE_IPS {
+    kind:             RunTimeHost
+    type:             dict[string, dict[string, string]]
+    standard:         true
+    since:            "0.4.13"
+    shortdoc:         "Partial traceroute IPs for configured TTLs"
```

Comment:
I'd suggest that this shortdoc should include the "why" or "what it means" more than the technical "what", maybe something like:
"Partial tracerouting for run-time environment discovery" ? I'm not sure what it should be, but I'd prefer any description to be more around its rationale for existing

Reply:
thanks. updated